aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,283 @@
+
+| Description            | Display                          | Display when space is limited   |
+| ---------------------- | -------------------------------- | ------------------------------- |
+| Within the last minute | Just now; a minute ago           | now; a min ago                  |
+| 1 day ago              | yesterday                        | yesterday                       |
+| < 1 week ago           | x days, hour, minute, second ago | x days, hr, min, sec ago        |
+| >= 1 week ago          | [Absolute time](#absolute-time)  | [Absolute time](#absolute-time) |
+| In 1 day               | tomorrow                         | tomorrow                        |
+| < 1 week from now      | in x days, hour, minute, second  | in x days, hr, min sec          |
+| In >= 1 week           | [Absolute time](#absolute-time)  | [Absolute time](#absolute-time) |
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Provide users with only relevant information. In this use case, users do not need to know how many seconds ago an activity was last performed or the exact time activities older than a month were performed.',
+    }}
+  >
+    <RelativeTimeExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't include additional information if it isn't needed. This will hurt readability and cause issues with layouts at smaller breakpoints.`,
+    }}
+  >
+    <RelativeTimeExample bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Abbreviating date and time
+
+When space in the UI is limited, units of time, days of the week, and months can be abbreviated.
+
+Some contexts in which abbreviations may be needed are:
+
+- Within cells of a data table
+- At smaller breakpoints
+
+| Units of time  | Abbreviation |
+| -------------- | ------------ |
+| millisecond(s) | ms           |
+| second(s)      | sec          |
+| minute(s)      | min          |
+| hour(s)        | hr           |
+| day(s)         | day, days    |
+| week(s)        | wk           |
+| month(s)       | mo           |
+| year(s)        | yr           |
+
+To abbreviate days of the week or months, use the [Javascript Intl.DateTimeFormat method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat). The general convention for English is to use the first 3 letters of the day or month name.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/date-and-time/AbbreviatedDateExample.js"
+  horizontalLayout
+  height="none"
+>
+  <AbbreviatedDateExample />
+</Example>
+
+## Alignment
+
+When presenting dates in a column, right-align [relative times](#relative-time) to bring units of the same magnitude into alignment.
```

Comment:
Should we mention that by default it should be left aligned? Maybe mention that when using things like ISO 8601 formatting, zero-padded dates, and generally when in comparison mode, it should be left aligned.

Might be helpful to have a similar example to what I had in the figma to show a table that might have both left and right alignment. Purpose of this example was to show the use case "trigger times" and "duration" as well. 

![image](https://github.com/grommet/hpe-design-system/assets/86075038/94599830-6987-478e-9c78-a9cd38598955)

Reply:
Done.